system-addon/lib/Store.jsm

```diff
@@ -76,10 +134,14 @@ this.Store = class Store {
    * @return {type}  description
```

Comment:
Looks like the UNINIT dispatching is gone, so updating the comment would be good.